### ISSUE TITLE
[Feature request] Ability to hide entries

### ISSUE THREAD

**Issue by ViBE-HU** [tiagoporsch/restartto#6]
Some entries maybe hardly ever used. It would be nice to hide them from the menu.

**Comment by Obi83** [comment]
maybe a small menu to choose which SSD/HDDs is showing in the restart to selection would be absolutly genius! i have 2x m.2 but its showing 5 drives to boot on. if i could change the showing drives instead of all, would be very appreciated. anyways, great tool mate!

**Comment by tiagoporsch** [comment]
I agree, and in fact I always had this idea in mind. I don't use GNOME anymore, but if someone wants to do this I'd happily review a PR.

**Comment by tiagoporsch** [comment]
Done in 9861dd43ff17dd1b7a9fea2a96f73b26f3d807ce.
